cmd/ocm/create/cluster/cmd.go

```diff
@@ -78,6 +79,14 @@ const clusterNameHelp = "will be used when generating a sub-domain for your clus
 
 const subnetTemplate = "%s (%s)"
 
+// the following regex defines five different patterns:
+// first pattern is to validate IPv4 address
+// second,is for IPv4 CIDR range validation
+// third pattern is to validate domains
+// and the forth petterrn is to be able to remove the existing no-proxy value by typing empty string ("").
+// nolint
+var UserNoProxyRE = regexp.MustCompile(`^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$|^(.?[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]$|^""$`)
```

Comment:
Yikes!
Instead of this monstrous incantation, could we instead use something like:
```go
func isValidNoProxy(val string) bool {
    return val==`""` || isIp(val) || isDomain(val) || isCIDR(val)
}
```
?

Reply:
This is how we defined it in CS as well as in ROSA. If we want to change that we may open a new ticket for that. This will require a comprehensive regression test too.